### PR TITLE
feat: migrated zkEVM testnet to zKyoto

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* feat: migrated zkEVM testnet to zKyoto
* feat: added zKatana bridge link (Gelato)

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Adds**

* feat: added zKatana bridge link (Gelato)
  * the link button is viewable when users connect to zKyoto

image
  
**Changes**

* feat: migrated zkEVM testnet to zKyoto

**Todos**
* update explorer URL to Startale's

### CODE REVIEW

File: src/components/bridge/BridgeSelection.vue [hunk 1]
```diff
@@ -185,20 +218,25 @@ import { useAccount, useNetworkInfo } from 'src/hooks';
 import { EthBridgeNetworkName } from 'src/modules/zk-evm-bridge';
 import { Path as RoutePath, buildEthereumBridgePageLink } from 'src/router/routes';
 import { computed, defineComponent } from 'vue';
-import { stargateUrl, layerSwapLink, relayBridgeAppLink } from 'src/modules/zk-evm-bridge/index';
+import {
+  stargateUrl,
+  layerSwapLink,
+  relayBridgeAppLink,
+  zkatanaBridgeUrl,
```
Comment (gluneau) [inline src/components/bridge/BridgeSelection.vue:225]: zKatana

File: src/components/bridge/BridgeSelection.vue [hunk 2]
```diff
@@ -224,6 +262,8 @@ export default defineComponent({
       buildEthereumBridgePageLink,
       stargateUrl,
       layerSwapLink,
+      zkatanaBridgeUrl,
```
Comment (gluneau) [inline src/components/bridge/BridgeSelection.vue:265]: zKatana

File: src/i18n/en-US/index.ts [hunk 3]
```diff
@@ -1002,6 +1002,11 @@ export default {
       text2: 'Available on Astar zkEVM. Switch the network to use it.',
       learn: '(learn more)',
     },
+    zkatanaBridge: {
```
Comment (gluneau) [inline src/i18n/en-US/index.ts:1005]: zKatana

File: src/modules/zk-evm-bridge/index.ts [hunk 4]
```diff
@@ -5,3 +5,4 @@ export const stargateUrl = 'https://stargate.finance/transfer';
 export const relayBridgeAppLink = 'https://relay.link/bridge/astar-zkevm/?fromChainId=1';
 export const layerSwapLink =
   'https://www.layerswap.io/app?to=astarzk_mainnet&from=ethereum_mainnet';
+export const zkatanaBridgeUrl = 'https://bridge.gelato.network/bridge/zkatana';
```
Comment (gluneau) [inline src/modules/zk-evm-bridge/index.ts:8]: zKatana

File: src/i18n/en-US/index.ts (hunk 3 above)
Comment (impelcrypto) [reply src/i18n/en-US/index.ts]: fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1238/commits/73807824eb5f82f09b41027309290152d704db13) commit

File: src/modules/zk-evm-bridge/index.ts (hunk 4 above)
Comment (impelcrypto) [reply src/modules/zk-evm-bridge/index.ts]: fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1238/commits/73807824eb5f82f09b41027309290152d704db13) commit

File: src/components/bridge/BridgeSelection.vue (hunk 2 above)
Comment (impelcrypto) [reply src/components/bridge/BridgeSelection.vue]: fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1238/commits/73807824eb5f82f09b41027309290152d704db13) commit

File: src/components/bridge/BridgeSelection.vue (hunk 1 above)
Comment (impelcrypto) [reply src/components/bridge/BridgeSelection.vue]: fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1238/commits/73807824eb5f82f09b41027309290152d704db13) commit

File: src/modules/zk-evm-bridge/index.ts [hunk 5]
```diff
@@ -5,3 +5,4 @@ export const stargateUrl = 'https://stargate.finance/transfer';
 export const relayBridgeAppLink = 'https://relay.link/bridge/astar-zkevm/?fromChainId=1';
 export const layerSwapLink =
   'https://www.layerswap.io/app?to=astarzk_mainnet&from=ethereum_mainnet';
+export const zKatanaBridgeUrl = 'https://bridge.gelato.network/bridge/zkatana';
```
Comment (gluneau) [inline src/modules/zk-evm-bridge/index.ts:8]: zKatana

File: src/modules/zk-evm-bridge/index.ts (hunk 4 above)
Comment (gluneau) [reply src/modules/zk-evm-bridge/index.ts]: That commit only capitalises the K in zKatana. 
That is not the commit you intended to submit I'm sure.

Comment (impelcrypto) [reply src/modules/zk-evm-bridge/index.ts]: I thought you asked just rename from 'zkatana' to 'zKatana'.

**Review by gluneau — APPROVED** [review]
Knowing we need the bridge to zKatana still, this is good for now.

**Review by bobo-k2 — APPROVED** [review]

**Pull request merged**